src/cast/Cast.js

```diff
@@ -243,8 +291,18 @@ function schemaToUplc(schema, x, defs = {}) {
                         )
 
                         if (ft) {
-                            const keyData = new ByteArrayData(encodeUtf8(key))
-                            const valueData = schemaToUplc(ft.type, value, defs)
+                            debugger
+                            //@ts-expect-error
+                            const rawKey = ft.tag || ft.name
```

Comment:
I'm a bit confused here, should we clean up `debugger` and `@ts-expect-error`? 
And `ft.tag` will always be `undefined` for `map` I think, only enum variants have tags (the constr index), so `ft.name` or `key` should suffice

Reply:
Yes, I'll get rid of the debugger before merging.

Ex:

```
struct cip68mapStruct {
  fname1: String
  taggedFname: Int "fn2"
}
```

I hadn't determined how to access both the defined field name (`taggedFname`) as well as the serialization format key (`fn2`).  AFAICT the previous version has only the serialization keys and doesn't allow `{fname1, taggedFname}` data.  I'd imagine `{fname1, fn}` would be allowable too (concerns welcome).

I've been a bit blocked on reaching this `debugger` statement ... will do.

Reply:
Currently only the tags can be accessed, and for fields without explicit tags the field name is used as an implicit tag: 
So here it would be: `{"fname1", "fn2"}`